pkg/controllers/management/clusterdeploy/clusterdeploy.go

```diff
@@ -531,7 +531,7 @@ func getAgentImages(name string) (string, string) {
 	return agentImages[nodeImage][name], agentImages[clusterImage][name]
 }
 
-func getCachedTaints(name string) []corev1.Taint {
+func getCachedControlPlaneTaints(name string) []corev1.Taint {
```

Comment:
nit: it would be nice to put a godoc comment here explaining what this function does, but as this is not a net-new function it is not a requirement.